### PR TITLE
retouch message

### PR DESCRIPTION
**Pull request by edgardoh**
Display a message when display mask is selected on retouch iop and blend module.

### CODE REVIEW

**Comment by TurboGit** [comment]
Sounds good, there is a missing message though. When clicking on the "display wavelet scale" button we should also have a message. This is not working and what I had reported before.

**Comment by edgardoh** [comment]
> When clicking on the "display wavelet scale" button we should also have a
> message.
>
what message?

**Comment by edgardoh** [comment]
I completely forgot about this. Done.

2018-09-13 9:21 GMT-03:00 Pascal Obry <notifications@github.com>:

File: src/iop/retouch.c [hunk 1]
```diff
@@ -2146,6 +2148,8 @@ static void rt_showmask_callback(GtkToggleButton *togglebutton, dt_iop_module_t
   // if blend module is displaying mask do not display it here
   if(module->request_mask_display && !g->mask_display)
   {
+    dt_control_log(_("display masks cannot be set when mask is being displayed on blend module"));
```
Comment (TurboGit) [inline src/iop/retouch.c]: maybe change this for consistency reasons with message above to:
"cannot display masks when the blending mask is displayed"

It is also shorter.

Comment (edgardoh) [reply src/iop/retouch.c]: Done.

**Comment by TurboGit** [comment]
All good, thanks.

**Comment by edgardoh** [comment]
BTW, based on the amount of feedback it seems that not too many people is
using it, is there a way to get more people involved in the testing?


2018-09-13 10:11 GMT-03:00 Pascal Obry <notifications@github.com>:

**Comment by TurboGit** [comment]
Send a message on the -dev (maybe -user) mailing list? We need people compiling dt at this stage.

**Comment by edgardoh** [comment]
OK, I'm already working on the documentation, so I can send a mail with
what I already have.

2018-09-13 11:36 GMT-03:00 Pascal Obry <notifications@github.com>:

**Pull request merged**